docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -1,41 +1,133 @@
-variable in `create-osx-user.sh` to a strong passphrase:
-    
 
---
-  
-. Download the scripts into the target macOS machine:
-+
 [source,bash]
 ----
-curl -O https://raw.githubusercontent.com/aerogear/digger-jenkins/master/admin/create-osx-user.sh
-curl -O https://raw.githubusercontent.com/aerogear/digger-jenkins/master/admin/enable-osx-remote-login.sh
-----
-+
-. Edit the `PASSWORD` for the Jenkins user in `create-osx-user.sh`.
-. Execute `create-osx-user.sh` with the updated password:
-+
-----
-sudo bash create-osx-user.sh
+#!/usr/bin/env bash
+
+##### I M P O R T A N T #####
+# This script helps with creating an OSX user.
+# Do not use this script blindly as it will allow access to your machine with default password.
+#
+# This script is not meant to be run in an automation. Run it manually.
+#
+# CHANGE the password below.
+# Also, make sure USER_ID below is not assigned.
+
+USERNAME="jenkins"
+PASSWORD="Password1"
+REAL_NAME="Jenkins Agent"
+GROUP_NAME="staff"
+
+# the first user's id is 500, second is 501 ...
+# picking a big number to be on the safe side.
+# You can run this one to list UIDs
+#   dscl . -list /Users UniqueID
+USER_ID=550
+
+# GID 20 is `staff`
+GROUP_ID=20
+
+
+###############
+############### end of parameters
+###############
+
+echo "IMPORTANT: this script will now create user ${USERNAME} with password ${PASSWORD} and add him to sudoers list."
+echo "Your machine will be accessible with SSH using these credentials."
+
+. /etc/rc.common
+dscl . create /Users/${USERNAME}
+dscl . create /Users/${USERNAME} RealName ${REAL_NAME}
+dscl . passwd /Users/${USERNAME} ${PASSWORD}
+
+dscl . create /Users/${USERNAME} UniqueID ${USER_ID}
+dscl . create /Users/${USERNAME} PrimaryGroupID ${GROUP_ID}
+dscl . create /Users/${USERNAME} UserShell /bin/bash
+dscl . create /Users/${USERNAME} NFSHomeDirectory /Users/${USERNAME}
+dseditgroup -o edit -a ${USERNAME} -t user admin
+cp -R /System/Library/User\ Template/English.lproj /Users/${USERNAME}
+chown -R ${USERNAME}:${GROUP_NAME} /Users/${USERNAME}
+
+echo "${USERNAME}  ALL=(ALL:ALL) ALL" >> /etc/sudoers
+
+echo "Done creating OSX user - you may need to restart the osx server to apply all changes for the user ${USERNAME}"
 ----
-+
-. Execute `enable-osx-remote-login.sh` to allow SSH access:
-+
+
+By default, this script will create a user named `jenkins` with SSH access. If using the above script you should read it carefully
+and understand its implications. The `PASSWORD` variable should be changed to something secure.
+
+To enable iOS builds, this user should be added to the `admin` group as in the above code snippet `dseditgroup -o edit -a ${USERNAME} -t user admin`
+
+This script, or individual commands to achieve the same state will need to be run on the macOS server. It can be downloaded for
+convenience using the following command: 
+
+`curl -O https://raw.githubusercontent.com/aerogear/digger-jenkins/master/admin/create-osx-user.sh`
```

Comment:
Can we put this above the script. Reading this first it seems like the user should copy the file. It's only clear at this point that the file can be downloaded.